src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -40,46 +41,58 @@ function getFlavor() {
 }
 
 function AboutPage(props) {
-    let popoverAnchor;
-    const menuItems = [
-        {
-            translationKey: 'initialSettingsPage.aboutPage.appDownloadLinks',
-            icon: Expensicons.Link,
-            action: () => {
-                Navigation.navigate(ROUTES.SETTINGS_APP_DOWNLOAD_LINKS);
+    const {translate, isShortcutsModalOpen} = props;
```

Comment:
We got rid of the `isShortcutsModalOpen` as per [your comment](https://github.com/Expensify/App/pull/28338#discussion_r1357458217). Can we remove this leftover?